addons/hr_expense/tests/common.py

```diff
@@ -55,6 +55,3 @@ def setUpClass(cls, chart_template_ref=None):
 
         # Ensure products can be expensed.
         (cls.product_a + cls.product_b).write({'can_be_expensed': True})
-        # Taxes on the products are included in price
-        (cls.product_a.supplier_taxes_id + cls.product_b.supplier_taxes_id).write({'price_include': True})
```

Comment:
To sum it up.. there are test that check account.move.line values but because of this it was not failing angel

As prior, we had domain and only allowed taxes with price_include: True; this made sense. But after we allowed all the taxes and force them to be price_include, I should have removed this. My bad.